src/components/smart_objects/include/smart_objects/object_schema_item.h

```diff
@@ -50,6 +50,12 @@ namespace ns_smart_objects {
  * @brief Object member.
  **/
 struct SMember {
+
+  enum class Type {
```

Comment:
@YarikMamykin use OEM_SPECIFIC and RPC_SPEC, and add doxygen description

Reply:
@LuxoftAKutsan fixed in e5df4ef22337bafe6ed7d68ff16c98b407f40d6c